pack.ps1

```diff
@@ -0,0 +1 @@
+dotnet pack -c Release .\MoreLinq\project.json
```

Comment:
Once more, switching to PowerShell scripts was not part of the plan for this PR (can be discussed/raised as a separate issue) so kindly revert to using `pack.cmd`. `pack.ps1` is adding no value at the moment (in fact, removes a lot) because in the end it's just calling `dotnet.exe` to do all the work.